.github/PROCESS.md

```diff
@@ -16,14 +16,35 @@ Keep guides short: we don't create rules; we automate.
 2. Pull request **title** should be accurate and descriptive as it is used in the generated changelog.
    It should not end with a dot.
    It should not mention the issue number but describe the fixed issue instead.
-3. Pull request **description** typically just mentions the issue number like "Refs #123." or "Closes #456.".
+3. Pull request **description** must mention the issue number like "Refs #123." or "Closes #456.".
    Do not use `https://github.com/org/repo/issue/123` even if it is rendered the same on GitHub.
+   We provide a pull requests template that includes suggestions and readiness checklist.
    End sentences with a dot.
 4. Commit messages (both titles and bodies) are not important and can be "WIP" or anything else.
    All commits are always squashed on merge by GitHub, so there is no need to squash them manually, amend them, and/or do force pushes.
-   Autogenerated GitHub's squash commit's title and body should be manually replaced by PR's title and description; that includes both manual merges and auto-merges.
+   Autogenerated GitHub's squash commit's title should be manually replaced by PR's title.
+   Autogenerated GitHub's squash commit's body should be manually replaced by "Refs #{issue_number}". 
    Do not leave original commit messages in the squashed commit body.
 
+## Draft pull requests
+
+1. We aim our issues to be scoped and well-described. 
+   With that, ideally, there shouldn't be a need to create a draft pull request.
```

Comment:
```suggestion
   With that, ideally, no need to create a draft pull request.
```

Reply:
> This suggestion is not grammatically correct (the verb is missing).

well, it's not clear then. 

The second sentence doesn't logically result from the first one. 

A draft is a draft and task decomposing is task decomposing. 

Why it should be connected somehow to our task decomposition?